modules/photons_products/base.py

```diff
@@ -114,13 +114,34 @@ def __call__(self, firmware_major, firmware_minor):
     def __repr__(self):
         return f"<Capability {self.product.name}>"
 
+    def __iter__(self):
+        caps = []
+        for cap_name, cap_value in list(self.items()):
+            if cap_name[:4] == "has_" and cap_value is True:
+                caps.append(f"{cap_name[4:]}")
+            elif cap_name[:4] == "has_" and cap_value is False:
+                caps.append(f"not_{cap_name[4:]}")
+            elif cap_name[:4] != "has_":
+                continue
+        return iter(sorted(caps))
+
     def __eq__(self, other):
         return (
             self.product == other.product
             and self.firmware_major == other.firmware_major
             and self.firmware_minor == other.firmware_minor
         )
 
+    def __contains__(self, item):
+        if type(item) is list:
+            return any(item) in self.__iter__()
```

Comment:
```suggestion
            return any(i in self for i in item)
```

also, do we want to support this?

Reply:
Yes, because it's required for the Filter of device finder so it can determine which devices have each provided cap, because the caps can be mutually exclusive, i.e. you can search for `matrix` and `multizone` and expect a result of Tiles, Candles, Beams and Zs.